docs/graphql/manual/schema/default-values/created-updated-timestamps.rst

```diff
@@ -0,0 +1,154 @@
+.. meta::
+   :description: Add created at / updated at timestamps
+   :keywords: hasura, docs, schema, default value, timestamps
+
+.. _created_updated_timestamps:
+
+Adding created_at / updated_at timestamps
+=========================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+
+We often need ``created_at`` and ``updated_at`` timestamp fields in our tables in order to indicate when an object was created or last updated.
+This page explains how to add these. 
+
+Add a created_at timestamp
+--------------------------
+
+.. rst-class:: api_tabs
+.. tabs::
+
+  .. tab:: Console
+
+    On the Hasura console, click on the ``Modify`` tab of a table. When clicking on the ``+Frequently used columns`` button, 
+    choose ``created_at``:
+
+    .. thumbnail:: /img/graphql/manual/schema/created-at.png
+        :alt: Add a created_at time on the Hasura console
+
+    Click the ``Add column`` button.
+
+  .. tab:: CLI
+
+    :ref:`Create a migration manually <manual_migrations>` and add the following SQL statement to add a ``created_at`` timestamp field to the ``article`` table:
+
+    .. code-block:: plpgsql
+
+      ALTER TABLE ONLY "public"."article" ADD COLUMN "created_at" TIMESTAMP DEFAULT NOW();
```

Comment:
Let's clarify that this goes in `up.sql` and include the content for `down.sql` as well.